### ISSUE TITLE
Change the upper limit of the high BPM stream SR bonus from 330 to 400

### ISSUE THREAD

**Issue by PPhenolOxidase** [ppy/osu#11132]
Currently, maps with streams above 330BPM are underrated in terms of SR. When the high BPM buff was implemented in Feb 2019, player skill was not at a high enough level to necessitate a higher BPM cap. However, a not insignificant amount of players are at a skill level where they can stream and burst at much higher BPMs. 

This would be a relatively simple change. All that needs to be done is change the cap to a higher BPM (or equivalent notes per second).

**Comment by PPhenolOxidase** [comment]
New to GitHub, this change was intended for Stable.

**Comment by bdach** [comment]
There's not really any difference when it comes to diffcalc, though. Both lazer and stable would need to facilitate this change, provided this bonus is in both.

**Comment by Cometblazer** [comment]
I'm of the mind that implementing something like this is necessary, but it's far too early to do it. From my 7-year experience as a player up to the modern day, burst streams over 300bpm are not really all that difficult; there's plenty of players I've seen that can play them properly. Longer streams and deathstreams on the other hand are notoriously difficult at ~270bpm, never mind 300bpm+.

Therefore, the amount of players who can play these sorts of maps constantly are in the mere dozens, which doesn't exactly warrant a cap increase *just yet.* I can reasonably see more justification with addressing this circa mid-late 2021, but as of the current moment, it seems trivial to add.

**Comment by PPhenolOxidase** [comment]
I totally agree that there are not too many players capable of deathstreaming 300+ BPM. However, there are some maps in the ranked section that are both underweighted in SR and have passes from multiple players (not just the typical high-ranked speed players). For example, [pishifat's difficulty on The Deceit](https://osu.ppy.sh/beatmapsets/991300#osu/2502889) has passes from over 30 players. 

Putting that aside however, I believe raising the speed SR cap is more a matter of principle than necessity. One of the mapsets I like to show people when discussing this is [Painters Gamma](https://osu.ppy.sh/beatmapsets/990177) with DT, where a 45 BPM increase (405 --> 450) only results in an SR increase of 0.46 (10.45 --> 10.91) Not only does this cap not result in an SR that accurately represents the difficulty of the maps, but it's actually less of an SR increase between the diffs than nomod (30 BPM increase, 0.97 SR increase). Furthermore, it would still make sense to increase the cap even if player skill has not yet reached the point where it is necessary. As you said, you can reasonably see justification addressing the cap in 2021. If there may be more of a reason to do it in the future, why not add it now instead of waiting until it becomes more of a problem than it already is? SR changes hardly affect the game as much as PP changes, and it would only affect a handful of maps on top of that.
